assembly/src/library/mod.rs

```diff
@@ -129,6 +130,20 @@ impl Module {
     pub fn write_source_locations<W: ByteWriter>(&self, target: &mut W) {
         self.ast.write_source_locations(target)
     }
+
+    pub fn write_into<W: ByteWriter>(&self, target: &mut W, options: AstSerdeOptions) {
+        self.path.write_into(target);
+        self.ast.write_into(target, options);
+    }
+
+    pub fn read_from<R: ByteReader>(
```

Comment:
minor nit: could we add short doc comments for these functions?

Reply:
Fixed.